### ISSUE TITLE
Complete App Heading Updates

### ISSUE THREAD

**Issue by bischoffa** [department-of-veterans-affairs/va-mobile-app#9111]
**Description:** 
In Q1 of 2024, an audit of the app headings was complete and an epic with recommended updates was issues. There have been many updates made to the app since that time. We should verify that all of the proposed updates were completed and that no additional changes are needed to the app headings with the recent changes to the app since that time.

**AC**
- [ ] Verify that all previously recommended app heading updates have been completed
- [ ] Perform a quick audit of the app to verify that no additional headings should be updated to align with accessibility best practices
